features/step_definitions/cg_worker_stepdef.rb

```diff
@@ -0,0 +1,53 @@
+
+# And(/^I click on child law services$/) do 
+#   page.execute_script("document.querySelector('#service_WPSLS-1-3').click()")
+# end
+
+# And(/^I click on court of protection services$/) do 
+#   page.execute_script("document.querySelector('#service_WPSLS-1-4').click()")
+# end
+
+# And(/^I click on Debt recovery services$/) do 
+#   page.execute_script("document.querySelector('#service_WPSLS-1-6').click()")
+# end
+
+# And(/^I click on Employment services$/) do 
+#   page.execute_script("document.querySelector('#service_WPSLS-1-12').click()")
+# end
+
+
+# And(/^I click on Contracts services$/) do 
+#   page.execute_script("document.querySelector('#service_WPSLS-2a-3').click()")
+# end
+
+When("I click on Litigation / dispute resolution") do
+  page.execute_script("document.querySelector('#service_WPSLS-1-10').click()")
```

Comment:
You don't need to use `execute_script` as capybara has inbuilt selector tools.
For example you could use:
```suggestion
  page.find('#service_WPSLS-1-10').click()
```